addons/l10n_gcc_invoice/models/account_move.py

```diff
@@ -92,20 +92,20 @@ def lang_product_name(line, lang):
             else:
                 line.l10n_gcc_line_name = line.name
 
-    def get_lines_grouped_by_section(self):
```

Comment:
Care that you can't remove a public function in stable :eyes: it's not stable friendly aha

Reply:
Let the function and add a comment saying she depreciated, like said before we should not delete function in stable version, if someone did a extend or an override it will traceback :smile: